src/scenes/BangBang/Game/Game.css

```diff
@@ -64,6 +65,30 @@
   object-fit: scale-down;
 }
 
+.wrong-local-container {
```

Comment:
Eu aprovei as mudanças, tá tudo certo, mas lembra quando for customizar uma classe dizer dentro de qual elemento (com id)

Igual na linha 40